sklearn/metrics/classification.py

```diff
@@ -577,7 +577,8 @@ class labels [2]_.
     return 1 - k
 
 
-def jaccard_similarity_score(y_true, y_pred, normalize=True,
+def jaccard_similarity_score(y_true, y_pred, labels=None, pos_label=1,
+                             average='samples', normalize='true-if-samples',
```

Comment:
just a note that this is not backward compatible with users calling it with positional arguments [sigh]! But I'm not sure what we should do in these cases.

Reply:
If we deprecate the current function and make jaccard_score that would solve it :)